src/main/java/com/teamc/fms/entity/SubscriptionType.java

```diff
@@ -0,0 +1,25 @@
+package com.teamc.fms.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Entity
+@Table(name = "subscription_type")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SubscriptionType {
+    @Id
+    private Integer id;
+    @NotNull
+    private String name;
+    @NotNull
+    private String description;
```

Comment:
description column is nullable
![image](https://github.com/web-development-course-way/fms/assets/31379298/414d1c8d-8060-479a-b0ed-794af45fef5e)